openstack/sharedfilesystems/v2/shares/results.go

```diff
@@ -110,3 +110,39 @@ type DeleteResult struct {
 type GetResult struct {
 	commonResult
 }
+
+// GrantAccessRes contains all information associated with an OpenStack share Grant Access
+type GrantAccessRes struct {
+	// The UUID of the share to which you are granted or denied access.
+	ShareID string `json:"share_id"`
+	// Timestamp when the share was created
+	CreatedAt time.Time `json:"created_at, omitempty"`
+	// Timestamp when the share was updated
+	UpdatedAt time.Time `json:"updated_at, omitempty"`
```

Comment:
Well, in case I have:
```
UpdatedAt time.Time json:"updated_at, omitempty"
```
I get an error:
```
syntax error: unexpected json, expecting semicolon, newline, or }
```
so I don't understand what you mean.